src/components/Output/custom-els/RangeInput/styles.css

```diff
@@ -0,0 +1,102 @@
+range-input {
+  position: relative;
+  display: flex;
+  height: 18px;
+  width: 130px;
+  margin: 2px;
+  font: inherit;
+  line-height: 16px;
+  overflow: visible;
+}
+
+range-input input {
+  position: relative;
+  flex: 1;
+  width: 100%;
+  padding: 0 !important;
+  margin: 0 !important;
+  background: none;
+  appearance: none;
+  -webkit-appearance: none;
+  outline-color: #2d9ac2;
+  outline-offset: 2px;
+}
+
+range-input input::-webkit-slider-runnable-track,
+range-input input::-moz-range-track,
+range-input input::-ms-track {
+  appearance: none;
+  -ms-appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+}
+
+range-input:before {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 8px;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  outline: none;
+  border: none;
+  background: #eee;
+  border-radius: 1px;
+  box-shadow: 0 -.5px 0 rgba(0,0,0,0.3), inset 0 .5px 0 rgba(255,255,255,0.2), 0 .5px 0 rgba(255,255,255,0.3);
+  background: linear-gradient(#34B9EB, #218ab1) 0/ var(--value-percent, 0%) 100% no-repeat #eee;
+}
+
+range-input input::-webkit-slider-thumb {
+  appearance: none;
+  -webkit-appearance: none;
+  background: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="10" height="10"><circle cx="5" cy="5" r="1" fill="#5D509E" /></svg>') center no-repeat #34B9EB;
+  box-sizing: content-box;
+  border-radius: 50%;
+  width: 12px;
+  height: 12px;
+  border: none;
+  box-shadow: 0 0.5px 2px rgba(0,0,0,0.3);
+  outline: none;
+}
+
+range-input input:focus::-webkit-slider-thumb {
+  box-shadow: 0 1px 3px rgba(0,0,0,0.5);
+}
+
+range-input > aside {
+  position: absolute;
+  left: 6px;
+  right: 6px;
+  bottom: 0;
+  height: 0;
+  overflow: visible;
+}
+
+range-input > aside > span {
+  background: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="32" height="62" fill="none"><path fill="#34B9EB" d="M27.3 27.3C25 29.6 17 35.8 17 43v3c0 3 2.5 5 3.2 5.8a6 6 0 1 1-8.5 0C12.6 51 15 49 15 46v-3c0-7.2-8-13.4-10.3-15.7A16 16 0 0 1 16 0a16 16 0 0 1 11.3 27.3z"/><circle cx="16" cy="56" r="1" fill="#5D509E"/></svg>') top center no-repeat;
+  position: absolute;
+  box-sizing: border-box;
+  left: var(--value-percent, 0%);
+  bottom: 3px;
+  width: 32px;
+  height: 62px;
+  text-align: center;
+  padding: 8px 3px 0;
+  margin: 0 0 0 -16px;
+  filter: drop-shadow(0 1px 3px rgba(0,0,0,0.3));
+  transform-origin: 50% 90%;
+  opacity: 0;
```

Comment:
I'm getting a bit of jank when the bubble first appears. I wonder if an opacity of 0.0001 and `will-change: transform` will improve responsiveness here.

Reply:
good call - I was seeing that but wasn't sure if it was related.

Reply:
that seems to have addressed it for me.

Reply:
... actually it seems like removing the focus ring addressed the issue. odd.

Reply:
Hah, I guess that's a heavy paint operation?